content/rs/administering/database-operations/database-backup.md

```diff
@@ -1,51 +1,118 @@
 future release.
 
-The backup process creates compressed (.gz) RDB files.
+The backup process creates compressed (.gz) RDB files that you can [import into a database]
+({{< relref "/rs/administering/database-operations/importing-data.md" >}}).
+If you backup a database configured for database clustering,
+RS copies a backup file for each shard to the specified backup location.
+
+{{% note %}}
+Make sure that you have enough space available in your storage location.
+If there is not enough space in the backup location, the backup fails.
+{{% /note %}}
+
+## Configuring Scheduled Backups
+
+To schedule backups for a database:
+
+1. Go to: **databases**
+1. Click on the database that you want to configure backups for.
+1. In **configuration**, select **Periodic backup**.
+1. Select an interval for the backups to run either every **1**, **4**, **12** or **24** hours.
+1. Select one of the available storage types.
+1. Enter the details for the selected storage type.
+
+### FTP server
+
+Before you configure backups to an FTP server, make sure that:
+
+- The RS instance has network connectivity to the FTP server.
+- The user that you specify in the FTP server location has read and write priviledges.
+
+To backup to an FTP server, enter the FTP server location in the format:
+
+```src
+ftp://user:password@host:port/path/
+```
+
+For example: `ftp://username:password@10.1.1.1:22/home/backups/
+
+### SFTP server
+
+Before you configure backups to an SFTP server, make sure that:
+
+- The RS instance has network connectivity to the SFTP server.
+- The user that you specify in the SFTP server location has read and write priviledges.
+- The RS server and SFTP server have the correct TLS certificates. You can select either:
+    - **Use the cluster auto generated key** - Go to settings and copy the **Cluster SSH Public Key**
+        to the SFTP server.
+    - **Use a custom key** - Generate a TLS key pair for the SFTP server and copy the private key to
+        the **SSH Private Key** box.
+
+To backup to an FTP server, enter the FTP server location in the format:
+
+```src
+sftp://user:password@host:port/path/
+```
+
+For example: `ftp://username:password@10.1.1.1:22/home/backups/
+
+### Amazon S3
+
+Before you configure backups to OpenStack Swift, make sure that you have:
+
+- Path in the format: `s3://bucketname/foldername/`
+- Access key ID
+- Secret access key
 
-**Note**: It is your responsibility to manage the backup location; Redis
-Enterprise Software does not delete old backups or monitor the amount of
-space left in the folder.
+### Local mount point
 
-### FTP location
+To backup to a local mount point:
 
-Backing up to FTP requires permissions to read and write to the FTP
-location. If needed, you can specify the username and password for the
-FTP location as part of the path with the following structure:
-ftp://user:password\@host:port/path/
+1. Connect to the terminal of the RS server.
```

Comment:
@Paz-Yanover You only need to configure on nodes that you want to backup, right? IOW you don't have to configure backups for all of the nodes.

Reply:
@bmansheim - backup is performed per database. Since shards can be located on every node, each node should have the relevant mount point/s.

Reply:
@Paz-Yanover If I configure periodic backups for a db and it has shards on multiple nodes, so I must configure the same mount point location (for example: /mnt/backups) and destination (for example: 192.168.10.204:/DataVolume/Public) on every node that has shards of this database?
It doesn't sound right to me because if a shard moves from a node that has periodic backups configured to a node that doesn't then the shard won't get backed up.